Update README.md

PAAMatias opened the pull request:
Created 'Development Environment Setup'  section

PAAMatias committed:
Update README.md

PAAMatias committed:
Updated REAME.md

Added some information to README.MD

PAAMatias committed:
Updated README.md

Added some information on the section "Development Environment Setup"

PAAMatias committed:
Added wp-content directory files

PAAMatias committed:
Updated README.MD

Added the 'Table of Contents', 'TODO' and 'Server Setup' sections and
updated the 'Development Environment Setup' section.

ruychi reviewed (changes requested):
No, no no...
## Development Environment Setup
Should recommend installing Vagrant, Virtualbox, Git, using Homebrew on macOS, Chocolatey on Windows, apt-get on Linux.

Step 2
Variable VVV can also be installed in macOS via Homebrew, so you do not need to manually add the PATH. On Windows....yeah, more manual work...
 
Step 4
after "vagrant up" you need to "vagrant ssh" to access the VM  

### Windows Suggestion
## Server Setup
Why installing Ubuntu?
It should be done through the Vagrantfile, i.e., "vagrant up" and then "vv create"

PAAMatias committed:
Eddited README.md

Edited the 'Development Environment Setup'

PAAMatias committed:
Eddited README.md

Added some information regarding macOS and Linux setup steps. Added the
Github sync steps.

PAAMatias committed:
Added website demo

PAAMatias committed:
Edited README.md

PAAMatias committed:
Added script to install Worpress

Added the script to install wordpress and all its dependencies on the remote servers.

PAAMatias committed:
Edited Readme.md
